dotcom-rendering/docs/architecture/029-signing-image-urls.md

```diff
@@ -1,3 +1,6 @@
+> **Warning**
```

Comment:
Should we put the status closer to the top, and update it to deprecated? Maybe pointing to another ADR / PR to what has usurped it?